cli/tests/services/logging_services.toml

```diff
@@ -0,0 +1,19 @@
+version = 1
+
+[services.one]
+command = '''
+i=1; while [ $i -lt 10 ]; do echo "$(i)"; sleep 0.1; i=$((i+1)); done
+'''
+
+[services.mostly-deterministic]
+command = '''
+echo 1
+echo 2
+echo 3
+
```

Comment:
nonblocking: there are a lot of very racey tests in both the integration and unit tests. I'm not sure if it's necessary, but it might be good to try to avoid the race. One idea is adding a `touch file` before the `sleep`, and then the tests can wait until that file exists

Reply:
And if the test failure was because of a race, we should definitely fix before merging
https://github.com/flox/flox/actions/runs/10302640067/job/28517184738?pr=1903#step:6:391

Reply:
fixed sync with @dcarley